src/test/python/replay_target.py

```diff
@@ -1,13 +1,24 @@
 import os
 import sys
-from Target import Target, ReplayTest
-from configEnum import buildSystem, TestResult, ReportType
+from Target import Target, ReplayTest, TargetRunProperties, ReplayTestOptions
+from configEnum import buildSystem, TestResult, ReportType, StartMode
 from markup_report_generator import Report_Generator
+from test_suite import ReplayTestSuite
 import subprocess
 
 def replay_target(target):
     # clone repo
-    os.system("git clone " + target.test_repo_url)
+    print(f"Starting replay test run for {target.test_repo_name} -> branch {target.target_run_properties.branch_name}")
+    clone_status = os.system(f"git clone -b {target.target_run_properties.branch_name} {target.test_repo_url}")
+
+    print(clone_status)
+    if clone_status == 32768:
+        os.system(f"cd {target.test_repo_name} && git checkout . && git checkout {target.target_run_properties.branch_name}")
```

Comment:
What does `git checkout .` do here?

Reply:
Reverts any changes made to current branch